trigger/task.go

```diff
@@ -469,7 +469,7 @@ func (t *taskTriggers) taskFailure(sub *event.Subscription) (*notification.Notif
 		return nil, nil
 	}
 
-	if !isFailedTaskStatus(t.data.Status) {
+	if !evergreen.IsFailedTaskStatus(t.data.Status) {
```

Comment:
Since `isFailedTaskStatus()` is unused, I would delete it.

Reply:
I want to call out Using IsFailedTaskStatus here instead of isFailedTaskStatus is effectively the same as what we discussed here https://github.com/evergreen-ci/evergreen/pull/5360#discussion_r798748663 (i.e. changing how the private function worked) but I do think this change may be fine; I would just check how often the new statuses are used (or if that was more legacy)